rebar.config.script

```diff
@@ -25,6 +25,7 @@ Profiles = {profiles,[ {escript, []}
                                                                 , getopt
                                                                 , gun
                                                                 , cowlib
+                                                                , jsx
```

Comment:
switch to 27 and use https://www.erlang.org/doc/apps/stdlib/json.html?

Reply:
true.

 json encoding with large payload is toooo slow.  Took me some time to find the slowness is coming from jsx. 
In my demo, or test report, I will use `eterm` encoding,  json encoding is reserved for specific customer.